core/domain/stats_domain.py

```diff
@@ -466,30 +471,116 @@ def validate(self):
                 self.classification_categorization)
 
 
+class AnswerOccurrence(object):
+    """Domain object that represents a specific answer that occurred some number
+    of times.
+    """
+    def __init__(self, answer, frequency):
+        """Initialize domain object for answer occurrences."""
+        self.answer = answer
+        self.frequency = frequency
+
+    def to_raw_type(self):
+        return {
+            'answer': self.answer,
+            'frequency': self.frequency
+        }
+
+    @classmethod
+    def from_raw_type(cls, answer_occurrence_dict):
+        return cls(
+            answer_occurrence_dict['answer'],
+            answer_occurrence_dict['frequency'])
+
+
+class AnswerFrequencyList(object):
```

Comment:
I think these domain objects should be subclasses of a (possibly trivial) superclass, just to make clear that they represent the same thing ("calculation output").

Reply:
Done.